### ISSUE TITLE
Allow users to specify the ID of statuses managed in Condition Lab

### ISSUE THREAD

**Issue by C-S-McFarland** [mclemente/fvtt-condition-lab-triggler#55]
I think this is related to #39.

When someone uses Condition Lab and adds new statuses, the statuses have a random string as their ID. I manage a module used with Lancer (https://foundryvtt.com/packages/csm-lancer-qol) and my mod adds some custom statuses. But it tries to detect if someone has added custom statuses to avoid a collision. When someone uses Condition Lab, I have to use the `name` field to match which status effects are registered in the config. This is a bad idea since names can be (should be?) localized, so matching on IDs is a better method for this. But Condition Lab has no way (that I can tell) to over-ride the id of the statuses added.

In short, my request is to add an option for each status added to over-ride the ID. For many users, this might never be used, but when there is a need to work with systems or other mods (like Lancer QoL or Midi QoL), it would be possible.
